INSS.EIIR.Web/Areas/Admin/Controllers/SubscriberController.cs

```diff
@@ -28,14 +37,89 @@ public IActionResult Index()
         }
 
         [Area(AreaNames.Admin)]
-        [Route(AreaNames.Admin + "/subscriber/{subscriberId}")]
+        [Route(AreaNames.Admin + "/subscriber/{subscriberId}", Name = "SubscriberProfile")]
         [Authorize(Roles = Role.Admin)]
         public async Task<IActionResult> Profile(int subscriberId)
         {
             var subscriber = await _subscriberDataProvider.GetSubscriberByIdAsync($"{subscriberId}");
```

Comment:
I thought we agreed new code would use the api's and not the data provider directly ?